### PR TITLE
Projects

### PR DESCRIPTION
**Pull request by andrzejkukuryk**
Projects section works.

### CODE REVIEW

File: src/scenes/projects/projectCardStack.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  stack: StackInfo[];
+}
+
+export function ProjectCardStack({ stack }: ProjectCardStackProps) {
+  return (
+    <div className={styles.container}>
+      <h5>Stack used:</h5>
+      <div>
+        {stack.map((icon) => (
+          <icon.component className={styles.stackIcon} />
```
Comment (anna-morawska) [inline src/scenes/projects/projectCardStack.tsx:16]: don't you get the missing key warning here?

File: src/scenes/projects/projects.tsx [hunk 2]
```diff
@@ -2,12 +2,22 @@ import React from "react";
...
   const { ref } = usePosition(updateProjectsPosition);
 
+
   return (
-    <section id="projects" ref={ref} className={styles.container}></section>
+    <section id="projects" ref={ref} className={styles.container}>
+      <div className={styles.innerContainer}>
+        <h2>Projects</h2>
+        {projects.map((project) => (
+          <ProjectCard project={project} />
```
Comment (anna-morawska) [inline src/scenes/projects/projects.tsx:18]: probably missing key warning

**Review by anna-morawska — APPROVED** [review]
Nice!

File: src/scenes/projects/projectCardStack.tsx (hunk 1 above)
Comment (andrzejkukuryk) [reply src/scenes/projects/projectCardStack.tsx:16]: I do ;) I'll fix it on the next branch "smallFixes" until tomorrow.

**Pull request merged**